pkg/serverless/apikey/env.go

```diff
@@ -66,6 +66,6 @@ func getSecretEnvVars(envVars []string, kmsFunc decryptFunc, smFunc decryptFunc)
 // their API key in plaintext through environment variables.
 func SetSecretsFromEnv(envVars []string) {
 	for envKey, envVal := range getSecretEnvVars(envVars, readAPIKeyFromKMS, readAPIKeyFromSecretsManager) {
-		os.Setenv(envKey, envVal)
+		os.Setenv(envKey, strings.TrimSpace(envVal))
```

Comment:
Just double checking for the sake of the comment on this function, but have we tested this with [dual shipping](https://docs.datadoghq.com/serverless/aws_lambda/configuration/?tab=datadogcli#send-telemetry-to-multiple-datadog-organizations)? Or are we pretty certain this won't affect that feature?

Reply:
Thanks for calling this out! I did some additional local testing with dual shipping and confirmed it's working as expected with a proxy (configured via environment variables and `datadog.yaml`) and without a proxy 👍 

I also updated one of the comments I changed to take into account the environment variables that can be set by dual shipping (`DD_ADDITIONAL_ENDPOINTS_SECRET_ARN`, `DD_ADDITIONAL_ENDPOINTS_KMS_ENCRYPTED`, etc.).